policies/bundles/TransportEncryptionAlgorithm/metric.rego

```diff
@@ -7,10 +7,12 @@ default compliant = false
 
 default applicable = false
 
+algorithm := endpoint.transportEncryption.algorithm
```

Comment:
Where comes the endpoint from? 
Shouldn't it be `input.httpEndpoint.transportEncryption.algorithm`?

Reply:
From the import in L4

Reply:
so true...